io/net/infiniband/dapl.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -e
+# This Program runs dapltest tests on client and server for the interfaces
+# specified in config file in the data directory. It runs for 5 different
+# inputs, as specified in multiplexer file.
+
+PATH=$(avocado "exec-path"):$PATH
+
+# Install dependencies
+if [[ `python -c 'from avocado.utils.software_manager import SoftwareManager; \
+   print SoftwareManager().install("dapl*")'` == 'False' ]]
+then
+    avocado_debug 'Dapl Packages not installed'
+    exit
+fi
+
+# Load Modules
+modules=(ib_uverbs ib_ucm ib_cm ib_mad ib_sa ib_umad ib_addr rdma_cm rdma_ucm \
+    ib_core mlx4_core mlx5_core mlx5_ib ib_ipoib mlx4_en mlx4_ib)
+for i in "${modules[@]}"; do
```

Comment:
How about having modules list taken from config, ignore if the list might not change at all.